package.json

```diff
@@ -10,6 +10,8 @@
     "run:build": "lerna run build --stream --scope",
     "build:invoices": "npm run run:build ui-invoices",
     "build:expenses": "npm run run:build ui-expenses",
+    "docker:invoices": "docker build -t invoices --build-arg app=invoices . --rm=true",
```

Comment:
Why don't you use `run:docker` ? (look previous line as an exemple)

Reply:
how you get param `app` at two emplacements ?